packages/wallet/backend/src/user/controller.ts

```diff
@@ -133,4 +134,24 @@ export class UserController implements IUserController {
       next(e)
     }
   }
+
+  verifyEmail = async (
```

Comment:
I think this should be moved into the auth module.

Reply:
moved controller method, but the service method remains in user service as all the work is with User model.